tests/worldcerealtests/test_preprocessing.py

```diff
@@ -28,3 +38,38 @@ def test_temporal_context_correction():
 
     # Should no longer raise an exception
     _validate_temporal_context(corrected_temporal_context)
+
+
+def test_worldcereal_preprocessed_inputs_graph():
+    """Test the worldcereal_preprocessed_inputs function.
+    This is based on constructing the openEO graph for the job
+    that would run, without actually running it."""
+
+    spatial_extent = geojson.loads(
```

Comment:
Should we have a geojson file in the testresources folder for this? I think it's more tidy

Reply:
Done in https://github.com/WorldCereal/worldcereal-classification/pull/153/commits/99d37c18bf42a63cc405becf89e69a02561d6af5